qemu/qemu.go

```diff
@@ -393,6 +413,10 @@ func (fsdev FSDevice) QemuParams(config *Config) []string {
 	fsParams = append(fsParams, fmt.Sprintf(",path=%s", fsdev.Path))
 	fsParams = append(fsParams, fmt.Sprintf(",security_model=%s", fsdev.SecurityModel))
 
+	if fsdev.Multidev != "" {
```

Comment:
Considering "warn" is the default behaviour, wouldn't "warn" be enforced anyways, making this check redundant?

Reply:
this new option was added in QEMU 4.2, so this check is needed to support versions less than 4.2

Reply:
I see, makes sense.